examples/help/help.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+
+	argparse "../.."
```

Comment:
Local imports are not allowed in Go (while they work, it is not allowed by standards). It should import package as `github.com/...`

Reply:
Without local imports, what do you recommend creating examples for changes before being accepted and merged? Should that be done after the changes are accepted?

Reply:
You can see how it is done in other examples, just importing `github.com/akamensky/argparse` will be enough. You can add new commits to PR by simply pushing more to your branch.